TBTK/calc/TightBindingLib/src/FileWriter.cpp

```diff
@@ -1146,4 +1146,97 @@ bool FileWriter::exists(){
 	return exists;
 }
 
+void FileWriter::writeParameterSet(
+    const Util::ParameterSet *parameterSet,
+    std::string name,
+    std::string path
+){
+	init();
+
+	const int ATTRIBUTES_RANK = 1;
+	const hsize_t limits_dims[1] = {1};
```

Comment:
attributes_dims instead of limits_dims. (I realize I have an inconsistent naming scheme using underscore instead of upper case letter here. Make underscore for now, I will change to upper case letters later.) Also, shouldn't the dimension be zero because we do not use the data section of the dataset? Maybe even the rank can be set to zero and the dimensions be set to NULL?